independent-publisher-connectors/OMDb API/apiDefinition.swagger.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "The OMDb API is a RESTful web service to obtain movie information, all content and images on the site are contributed and maintained by our users.\nThis API requires authorization, you can get a free key here: [http://omdbapi.com/apikey.aspx](http://omdbapi.com/apikey.aspx)",
```

Comment:
Please remove the API words from the description. and please remove the link to obtain a free key and you can add this to the readme file under "Obtaining Credentials" section. 
Please follow [these requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) for the description.